nikolai4D/ContechOS
# C2 165 filter box ascending alphabetic order on nodes

## Description & motivation


The nodes in the filter box where unsorted.
They are now sorted by a simple function on top level and children level.


## How to test



1. Go to the filter view
2. In the filter box, check the alphabetic order of the nodes on top level
3. Click on one checkbox to reveal children, and see their alphabetic order 

## New tickets to be added



None

---
